### PR TITLE
Update ediscovery.md

### PR DESCRIPTION
**Pull request by msbemba**
Added 
- Exchange online Plan 2
- Sharepoint online Plan 2
As customer can use the standard feature with these stand alone licenses as well without having to have the entire E3 sub

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8812

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/ediscovery.md [hunk 1]
```diff
@@ -140,6 +140,8 @@ The following sections show the minimum subscription requirements for Content se
 
 ### eDiscovery (Standard)
 
+- Exchange online Plan 2
+- Sharepoint online Plan 2
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery.md:144]:
```suggestion
- Exchange Online Plan 2
- SharePoint Online Plan 2
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy reciew.

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**